libindy_vdr/src/ledger/requests/cred_def.rs

```diff
@@ -53,6 +53,14 @@ impl CredentialDefinition {
     }
 }
 
+impl From<CredentialDefinition> for CredentialDefinitionV1 {
+    fn from(cred_def: CredentialDefinition) -> Self {
+        match cred_def {
+            CredentialDefinition::CredentialDefinitionV1(cred_def) => cred_def
+        }
+    }
+}
+
```

Comment:
I know this is used a lot in the anoncreds code but it's a bad pattern which is why I removed it. In indy-credx it does the downcasting in each place that the type is used. I can merge it if that's needed though.

Reply:
I agree. I'll do the same for Libindy.